crates/ruff_linter/src/fix/edits.rs

```diff
@@ -122,6 +122,28 @@ pub(crate) fn remove_unused_imports<'a>(
     }
 }
 
+/// Edits to make the specified imports explicit, e.g. change `import x` to `import x as x`.
+pub(crate) fn make_redundant_alias<'a>(
+    member_names: impl Iterator<Item = &'a str>,
+    stmt: &Stmt,
```

Comment:
I'm not actually sure what your suggestion is here, probably because I don't know how to get from what I have in the caller (a `NodeId`) to an `AnyImport`. I was following existing code nearby which called `Semantic::statement` to convert `NodeId` to `Stmt`. What is the benefit of using `AnyImport`?

Reply:
(Ah yeah, I don't think `AnyImport` does what you want. `AnyImport` is for a binding, but you're operating on an entire statement here. You actually have an `AnyImport` already via `let Some(import) = binding.as_any_import()`.)

Reply:
I just saw that in the caller I have `ImportBinding.import : AnyImport`.

And yes, this code is producing an edit for a statement that incises one or more bindings, so the comment isn't applicable.